exotica_python/package.xml

```diff
@@ -14,4 +14,5 @@
   <depend>shape_msgs</depend>
   <depend>python-matplotlib</depend>
   <depend>python-pyassimp</depend>
+  <depend>python-Tkinter</depend>
```

Comment:
No, the correct key is `python-tk`, cf. https://github.com/ros/rosdistro/blob/61959ac7a9d2d2bc91c0555ce139c22de428a917/rosdep/python.yaml#L4255

Reply:
resolved, also added `python-rospkg` as per [here](http://docs.ros.org/jade/api/catkin/html/howto/format1/python_module_dependencies.html)